File: Array-Bit-Manipulation/260-single-number-iii/single-number-iii.cpp
class Solution {
public:
    vector<int> singleNumber(vector<int>& nums) {
        vector<int> res;
        int n = size(nums);
        for (int i = 0; i < n; ++i) {
            bool ifDuplicate = false;
            for (int j = -0; j < n; ++j) {
                if (i != j && nums[i] == nums[j]) {
                    ifDuplicate = true;
                    break;
                }
            }
            if (!ifDuplicate) {
                res.push_back(nums[i]);
            }
        }
        return res;
    }
};

File: Array-Bit-Manipulation/260-single-number-iii/single-number-iii.php
class Solution {
    public function singleNumber($nums) {
        $res = [];
        $n = count($nums);
        for ($i = 0; $i < $n; ++$i) {
            $ifDuplicate = false;
            for ($j = 0; $j < $n; ++$j) {
                if ($i != $j && $nums[$i] == $nums[$j]) {
                    $ifDuplicate = true;
                    break;
                }
            }
            if (!$ifDuplicate) {
                $res[] = $nums[$i];
            }
        }
        return $res;
    }
}